modin/core/dataframe/pandas/dataframe/utils.py

```diff
@@ -0,0 +1,266 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Collection of algebra utility functions, used to shuffle data across partitions."""
+
+import numpy as np
+import pandas
+from typing import Callable, Union, Optional
+from collections import namedtuple
+
+from modin.core.dataframe.pandas.dataframe.dataframe import PandasDataframe
+
+
+SortFunctions = namedtuple(
```

Comment:
I wonder if that name is somewhat wrong, maybe it should be `ReshuffleFunctions` or smth? 'cause it could be used for other means aside of sorting where we need to move data between partitions to make things efficient

Reply:
Makes sense!